spec/support/test_workers.rb

```diff
@@ -40,3 +40,21 @@ class BatchedUniqueArgsWorker
   def perform(foo)
   end
 end
+
+class BatchedAtOnceWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :batched_at_once, batch_size: 3, batch_at_once: true
+
+  def perform(foo)
+  end
+end
+
+class BatchedAtOnceIntervalWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :batched_at_once_interval, batch_size: 3, batch_at_once: true, batch_flush_interval: 3600
+
+  def perform(foo)
```

Comment:
Unused method argument - `foo`. If it's necessary, use `_` or `_foo` as an argument name to indicate that it won't be used. You can also write as `perform(*)` if you want the method to accept any arguments but don't care about them.